README.md

```diff
@@ -10,20 +10,6 @@ The main objectives of the workshop are to:
 - learn how to serve dynamic files using Handlebars.js templates
 - learn how to use Handlebars views, layouts and partials
 - be introduced to Handlebars helpers
-- there is an additional readme ([mentors-notes.md](./mentors-notes.md)) that the mentor can use to follow step by step instructions for the code-along workshop.
-
-## Model-View-Controller (MVC) model
```

Comment:
What is the motivation for getting rid of this?

Reply:
Previously there were only mentors notes and no other instructions. The mentors notes were brief, and did not include detailed instructions. They are now unnecessary as we have written detailed step by step instructions.
If you are referring to the removal of the MVC section in the readme, see #2 . Personally I don't think it added anything to this workshop, and there is plenty in there already.